drivers/user_directory_report/app/views/user_directory_report/warehouse_reports/users/report.haml

```diff
@@ -0,0 +1,25 @@
+- content_for :title, _('User Directory')
+
+= render 'warehouse_reports/breadcrumbs', tab: 'operational'
+
+.o-page
+  .o-page__header
+    - @prompt = 'Search Users'
+
+    .o-page__controls
+      .o-page__search
+        = render 'search_form', tooltip: 'Search on a user\'s first name, last name or email address', aria: { label: 'User Search' }, data: {placement: 'bottom'}
+
+.row
+  .col-sm-8
+    %h1= content_for :title
+
+- if cas_available?
+  %ul.nav.nav-tabs{role: :tablist}
+    %li.nav-item{role: :presentation}
+      = link_to 'Warehouse Users', 'warehouse', class: nav_link_classes('warehouse', @user_source)
+    %li.nav-item{role: :presentation}
+      = link_to 'Cas Users', 'cas', class: nav_link_classes('cas', @user_source)
```

Comment:
Generally the second argument should be a path helper, something like `user_directory_report_warehouse_reports_users_warehouse_path` (I have no idea if that's the right path.)  A simple string might work here, but if we use the path helper, then we could pass the search params along as well by using the `link_params` method in ApplicationController.  That would allow for maintaining the previous search when going between warehouse and CAS tabs.